home/index.html

```diff
@@ -105,6 +101,11 @@ <h2 class="section" id="overview">Features</h2>
 <p>
   This build file also includes a directive to upload your artifacts to Bintray automatically.
 </p>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Fast builds</h2>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Intuitive plug-in architecture</li></h2>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Half the size of Gradle</h2>
```

Comment:
I don't think we should say it's half the size of Gradle: users won't care about this and who knows, Kobalt might end up being pretty big too...